content/user/troubleshooting/blank_page.md

```diff
@@ -71,8 +71,9 @@ The most common customization errors include:
 *   adding or deleting apostrophes or quotations (ie: ' or " marks) inside define() statements, resulting in mismatched quotes
 *   removing or adding punctuation to define() statements, thus breaking the correct syntax. A define() statement should look like this:
 
-
-`define('CONSTANT_NAME', 'value here');`
+```
+    define('CONSTANT_NAME', 'value here');
+```
 
 *   missing semi-colons and periods, unbalanced parentheses, braces, brackets
 *   uploading files to the wrong places. For more information about overrides, see the [Overrides Chart](http://www.zen-cart.com/index.php?main_page=product_contrib_info&cPath=40_54&products_id=298) and the [Overrides FAQs](http://tutorials.zen-cart.com/index.php?category=4)
```

Comment:
Perhaps these links can stand an update too.

Reply:
Thanks for catching - fixed in upcoming PR.